.changeset/sharp-tables-change.md

```diff
@@ -0,0 +1,6 @@
+---
+"@chakra-ui/layout": patch
+"@chakra-ui/theme": patch
```

Comment:
Please use a minor bump for both packages, because this PR adds a feature. Adding something increments in [semver](https://semver.org/) terms the minor version.

```
"@chakra-ui/layout": minor
"@chakra-ui/theme": minor
```

Reply:
Changed to "minor"